dirty_cat/super_vectorizer.py

```diff
@@ -110,11 +104,12 @@ class SuperVectorizer(ColumnTransformer):
         If set to `True`, will try to convert each column to the best possible
         data type (dtype).
 
-    handle_missing: str, default=''
-        One of the following values: 'error' or '' (empty).
-        Defines how the encoder will handle missing values.
-        If set to 'error', will raise ValueError.
-        If set to '', will impute the missing values (pd.NA) with blank strings.
+    impute_missing: str, default='auto'
+        By-column missing values imputation parameter.
+        'auto' will impute missing values if it's considered appropriate.
+        'force' will impute all missing values.
+        'skip' will not impute at all.
```

Comment:
The render isn't so bad, but not too great either.

![image](https://user-images.githubusercontent.com/33222654/131847463-c00c0df0-ba3d-4ac0-9b6c-f9939408b09d.png)
https://1040-124964482-gh.circle-artifacts.com/0/doc/generated/dirty_cat.SuperVectorizer.html#dirty_cat.SuperVectorizer

It's readable but I agree it could use some better style :)